iwyu_output.h

```diff
@@ -175,6 +176,23 @@ class OneIncludeOrForwardDeclareLine {
   const clang::NamedDecl* fwd_decl_;
 };
 
+class UsingDeclStatus {
```

Comment:
I don't mind this class as much now that it's disassociated from the other `One...` classes.

But I still find it lacks cohesion, see below.